packages/workers-playground/src/QuickEditor/MigrateToWrangler/DocsLink.tsx

```diff
@@ -0,0 +1,42 @@
+import { Icon } from "@cloudflare/component-icon";
+import { A, Span } from "@cloudflare/elements";
+import { isDarkMode } from "@cloudflare/style-const";
+import { createStyledComponent } from "@cloudflare/style-container";
+import React from "react";
+
+const Link = createStyledComponent(
+	({ theme }) => ({
+		display: "inline-flex",
+		alignItems: "center",
+		width: "fit-content",
+		fontSize: theme.fontSizes[1],
+		color: `${theme.colors.indigo[isDarkMode() ? 9 : 2]} !important`,
+		backgroundColor: theme.colors.indigo[isDarkMode() ? 4 : 9],
+		borderRadius: "100vw",
+		px: 2,
```

Comment:
What unit is `2`? Is it rems?

Reply:
No, it's a unit in our design system, which ends up rendered as `8px`